pkg/collector/process_test.go

```diff
@@ -366,3 +381,15 @@ func TestUDPCollectingProcess_TemplateExpire(t *testing.T) {
 	assert.Nil(t, template, "Template should be deleted after 5 seconds.")
 	assert.NotNil(t, err, "Template should be deleted after 5 seconds.")
 }
+
+func waitForCollectorReady(t *testing.T, address net.Addr) {
+	checkConn := func() (bool, error) {
+		if _, err := net.Dial(address.Network(), address.String()); err != nil {
+			return false, err
+		}
+		return true, nil
+	}
+	if err := wait.Poll(time.Millisecond, time.Millisecond, checkConn); err != nil {
```

Comment:
increase the poll interval to 100 milliseconds and timeout to 500 milliseconds. One millisecond is too aggressive.